src/backend/distributed/sql/udfs/citus_finish_pg_upgrade/latest.sql

```diff
@@ -107,6 +107,8 @@ BEGIN
     FROM
         old_records naming,
         pg_get_object_address(naming.type, naming.object_names, naming.object_args) address;
+
+#include "../../../../columnar/sql/udfs/citus_finish_pg_upgrade/10.0-1.sql"
```

Comment:
I feel like we'll regret this one day, but ok.

Reply:
depending on what we will regret there are options to address:

 - the `../../../../` we can prevent by changing the flags to the preprocessor to set the searchpath to `src/backend`. In that case we could omit the 4 directories up part, and address the file by `columnar/sql/udfs/citus_finish_pg_upgrade/10.0-1.sql`
 - including a snippet of other files here. We can define an internal UDF for ensuring the access methods are created. This might be something we want to do already based on your other comment on the file being included here.

I will change this include into a function call. However this will add an other internal udf which increases the surface of functions we expose.